lib/stripEndings.js

```diff
@@ -0,0 +1,52 @@
+const toUnicode = require( './toUnicode' )
+const toAscii = require( './toAscii' )
+const { getRegexClass, getRegexGroup } = require( './regex-utils' )
+
+// Line endings in both ASCII, Unicode, and English
+const endingClass = getRegexClass( [ '।', '॥', ']', '[', '|' ] )
+// Sometimes translation line endings begin with these characters, before numbers
+const optionalEndingClass = getRegexClass( [ '(' ] )
+// Remove any broken endings
+const brokenEndingClass = getRegexGroup( [ '()' ] )
+
+// All numbers in ASCII, Unicode
+const numberClass = getRegexClass( [
+  ...Array.from( { length: 10 }, ( _, i ) => i.toString() ),
+  ...Array.from( { length: 10 }, ( _, i ) => toUnicode( i.toString() ) ),
+] )
```

Comment:
Wouldn't it be cheaper and easier to understand if you just made an array / mapping const variable?

Reply:
It’s negligible, since this is done once on module load, and not per function call.

Reply:
Okay but reading wise [1, 2, 3, 4, 5, 6, 7, 8, 9, 0]  is shorter and easier to read lol

Reply:
LOL I was going to suggest that `Array.from` was, but even I know I can't really win that one 😂😂